package.json

```diff
@@ -23,7 +23,7 @@
     "preversion": "./scripts/version.sh --pre",
     "version": "./scripts/version.sh",
     "postversion": "./scripts/version.sh --post",
-    "test": "jest"
+    "test": "jest && ./test/jest/scripts/post-test.sh"
```

Comment:
@brainthinks what if I want jest to watch my changes? Can I still use this?

Reply:
or are we going to use another script command for developer use and use this one only for ci.

Reply:
is setting up a watcher for testing a thing?

Reply:
add `test:watch` script for people who want to watch their tests.

Reply:
add `test:min` or `test:dev` some script in which developers would use that would be FAST for them to get feedback. As code coverage is not something a developer may care about seeing when working on a bug or feature right away.

Reply:
added